src/OpenLoco/src/Paint/PaintRoadStyle0Data.h

```diff
@@ -0,0 +1,195 @@
+#pragma once
+#include "Objects/RoadObject.h"
+#include "Paint.h"
+#include "PaintRoadCommonData.h"
+#include <OpenLoco/Core/Numerics.hpp>
+#include <array>
+#include <span>
+
+namespace OpenLoco::Paint::Style0
+{
+    // 0x004083B1, 0x004084BC, 0x004083B1, 0x004084BC
+    constexpr RoadPaintMergeablePiece kStraight0 = {
+        std::array<uint32_t, 4>{ 3486, 3487, 3486, 3487 },
+        /* StreetlightHeights */ kNoStreetlights,
+        /* IsMultiTileMerge */ true,
```

Comment:
Given we've been using C++20 for a while, can't we just use designated initialisers?

Reply:
Yes we should do that.

Reply:
No can't do that due to using a constructor